refact_webgui/webgui/tab_server_logs.py

```diff
@@ -48,7 +74,28 @@ async def get_log(right_file):
             return Response("File \"%s\" not found\n" % log_name, media_type="text/plain")
         right_file = list_of_files[-1]
 
+        async def streamer() -> AsyncIterator[str]:
+            last_n_lines = 10_000
+
+            if (lines_cnt := await count_lines_wc(right_file)) > last_n_lines:
+                yield f'{lines_cnt - last_n_lines} lines were skipped...\n'
+
+            async with aiofiles.open(right_file, 'r') as f:
+                # going to the end of the file before streaming lines_from_tail -- while we are streaming some new lines might appear
+                await f.seek(0, os.SEEK_END)
```

Comment:
what is the purpose of this line? why we should go to the end of the file if we simply need to stream last n lines?